src/static/css/skip-nav.less

```diff
@@ -0,0 +1,24 @@
+#skip-nav {
```

Comment:
I'll try it out to see how it looks. The problem is that right now the skip nav link appears right over the yellow beta banner and there aren't many colors that look good on that.